src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -417,5 +417,16 @@ def tab_link(label, path, active = false)
 
     link_to(label, path, class: html_class)
   end
+
+  def webui2_user_or_group_image_tag(user_or_group:, css_class: nil, alt: nil, title: nil, size: 500)
```

Comment:
why do you need a custom title? I would try to generalize instead... (similar to `alt`)